EKF/control.cpp

```diff
@@ -1071,6 +1071,13 @@ void Ekf::controlHeightFusion()
 	}
 
 	if (fuse_height) {
+
+		// Reset all height tests ratio
+		_gps_pos_test_ratio(1) = 0;
+		_ev_pos_test_ratio(1) = 0;
+		_baro_hgt_test_ratio(1) = 0;
+		_rng_hgt_test_ratio(1) = 0;
```

Comment:
```suggestion
		resetHeightTestRatios();
```
Could you please put them in a small function like that?